docs/bill-of-materials.csv

```diff
@@ -0,0 +1,20 @@
+"J2","USB_C_Receptacle_USB2.0_16P","https://www.lcsc.com/datasheet/lcsc_datasheet_2312191005_G-Switch-GT-USB-7047A_C963216.pdf","BadgeMagic:USB_C_Receptacle_GCT_USB4110","1","0.1577"," GT-USB-7047A","USB 2.0-only, 16P, SMD, Type-C Receptacle connector. There are 2 mechanical pin for positioning, should be chop off before soldering."
+"KEY1,KEY2","button","https://www.lcsc.com/datasheet/lcsc_datasheet_1811011023_HYP--Hongyuan-Precision-1TS003A-1800-3500A-CT_C255806.pdf","BadgeMagic:1TS003A-1800-3500A-CT","2","0.0431"," 1TS003A-1800-3500A-CT","SPST SMD,2x4mm Tactile Switches"
+"MK1"," WM-64C","https://www.lcsc.com/datasheet/lcsc_datasheet_2312271240_PANASONIC-WM-64C_C19458520.pdf; https://datasheet.octopart.com/WM-64PCX-Panasonic-datasheet-13695803.pdf","BadgeMagic:WM-64PC","1","2.0725"," WM-64C","Electret microphone"
```

Comment:
Please remove spacing in front of WM-64C

Reply:
Resolved